src/EditorFeatures/CSharpTest/Formatting/FormattingEngineTests.cs

```diff
@@ -1970,6 +1970,41 @@ void MyMethod()
             await AssertFormatWithBaseIndentAsync(expected, code, baseIndentation: 4);
         }
 
+        [WpfFact]
+        [Trait(Traits.Feature, Traits.Features.Formatting)]
+        [WorkItem(27268, "https://github.com/dotnet/roslyn/issues/27268")]
+        public async Task PositionalPattern()
+        {
+            var code = @"[|
+class MyClass
+{
+    void MyMethod()
+    {
+        var point = new Point (3, 4);
+        if (point is Point (3, 4) _
+            && point is Point{x: 3, y: 4} _)
+        {
+        }
+    }
+}
+|]";
+            var expected = @"
+class MyClass
+{
+    void MyMethod()
+    {
+        var point = new Point(3, 4);
+        if (point is Point(3, 4) _
+            && point is Point { x: 3, y: 4 } _)
+        {
+        }
+    }
+}
+";
+
+            await AssertFormatWithBaseIndentAsync(expected, code, baseIndentation: 4);
+        }
```

Comment:
>} [](start = 8, length = 1)

Would it be useful to also test different options, since it looks like positional patterns will be formatted like methods, and methods are governed by certain options?

Reply:
Tests updated